<launch>

	<group ns="image_pub">
		<node pkg="image_processing_hw" name="image_pub" type="image_pub_all.py"/>
	</group>

	<group ns="homework7">
		<node pkg="hw2_package" name="image_processing" type="imaging.py"/>

		<node pkg="rqt_graph" name="rqt_graph" type="rqt_graph" />
		<node pkg="rqt_console" name="rqt_console" type="rqt_console" />

		<node pkg="rqt_image_view" name="image_view_cropped" type="rqt_image_view" args="/image_cropped"/>
		<node pkg="rqt_image_view" name="image_view_white" type="rqt_image_view" args="/image_white"/>
		<node pkg="rqt_image_view" name="image_view_yellow" type="rqt_image_view" args="/image_yellow"/>
	</group>
	<node pkg="rostopic" name="rostopic" type="rostopic" output="screen" args="echo /image"/>
</launch>